### ISSUE TITLE
SiteConfig Column 'ID' in where clause is ambiguous when applying FluentExtension

### ISSUE THREAD

**Issue by mhagemann29** [tractorcow/silverstripe-fluent#331]
With silverstripe 4.0.1 and silverstripe-fluent 4.0.0-beta2 

YAML configuration:

```
Silverstripe\SiteConfig\SiteConfig:
  extensions:
    - TractorCow\Fluent\Extension\FluentExtension
```

Error message:

```
[Emergency] Uncaught SilverStripe\ORM\Connect\DatabaseException: Couldn't run query:

SELECT DISTINCT "SiteConfig"."ClassName", "SiteConfig"."LastEdited", "SiteConfig"."Created", CASE
	WHEN "SiteConfig_Localised_en_US"."ID" IS NOT NULL THEN "SiteConfig_Localised_en_US"."Title"
	ELSE "SiteConfig"."Title" END
 AS "Title", CASE
	WHEN "SiteConfig_Localised_en_US"."ID" IS NOT NULL THEN "SiteConfig_Localised_en_US"."Tagline"
	ELSE "SiteConfig"."Tagline" END
 AS "Tagline", "SiteConfig"."CanViewType", "SiteConfig"."CanEditType", "SiteConfig"."CanCreateTopLevelType", "SiteConfig"."ID", 
			CASE WHEN "SiteConfig"."ClassName" IS NOT NULL THEN "SiteConfig"."ClassName"
			ELSE 'SilverStripe\\SiteConfig\\SiteConfig' END AS "RecordClassName", 'en_US' AS "Locale"

FROM "SiteConfig"
LEFT JOIN "SiteConfig_Localised" AS "SiteConfig_Localised_en_US" ON "SiteConfig"."ID" = "SiteConfig_Localised_en_US"."RecordID" AND "SiteConfig_Localised_en_US"."Locale" = ?

WHERE (ID = ?)

LIMIT 1

23000-1052: Column 'ID' in where clause is ambiguous
```

Let me know if anything else is needed.

**Comment by robbieaverill** [comment]
I think there was a core issue that is probably the same cause @tractorcow relating to postgresql?

**Comment by tractorcow** [comment]
This is a siteconfig issue and will be fixed in 4.0.2 (of siteconfig, not fluent).

**Comment by tractorcow** [comment]
https://github.com/silverstripe/silverstripe-siteconfig/pull/87
